board/safety.h

```diff
@@ -500,7 +504,7 @@ int ROUND(float val) {
 
 // Safety checks for longitudinal actuation
 bool longitudinal_accel_checks(int desired_accel, const LongitudinalLimits limits) {
-  bool accel_valid = get_longitudinal_allowed() && !max_limit_check(desired_accel, limits.max_accel, limits.min_accel);
+  bool accel_valid = get_accel_allowed() && !max_limit_check(desired_accel, limits.max_accel, limits.min_accel);
```

Comment:
If you enable at a stop with foot on brake, we treat this state as pre-enabled in openpilot. We want to block just gas in this state, since requesting brake to stay stopped is fine (and needed for some brake/non-accel cars like GM and Honda), and we switched to allowing steering in pre-enable as well.

Currently, this should block those messages. Can we change this to allow only brake?